api/handler/acl.go

```diff
@@ -345,6 +353,10 @@ func (h *handler) PutObjectACLHandler(w http.ResponseWriter, r *http.Request) {
 		return
```

Comment:
If `getBucketAndCheckOwner` was successful and `GetObjectInfo` wasn't the gw couldn't log at least`oid`.

Reply:
If an error takes place and objInfo is `nil` then the handler will call logAndSendError and return. The log won't happen plus we will have details on the problem.

Reply:
Sorry, it was a typo. Fixed:
If getBucketAndCheckOwner was successful and GetObjectInfo wasn't the gw couldn't log at least `cid`.

Reply:
1) the logging is gone from some handlers now: Denis suggested moving it into a couple of internal methods.
2) There are other places where your comment is applicable though. And again, `logAndSendError` saves us there: if we run into errors before the log.Debug gets called then the needed details will be in error logs. However, for cases of valid CID+OID we will have our Debug log line in case of errors that happen further.

Reply:
> We already log CID of bucket on creating, and can get it using HeadBucket request

As for me, it doesn't seem convenient